schema/cassandra/cadence/versioned/v0.39/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "CurrVersion": "0.39",
+  "MinCompatibleVersion": "0.39",
+  "Description": "Adding create time and last update time for history tables",
+  "SchemaUpdateCqlFiles": [
+    "task_list_partition_config.cql"
```

Comment:
so, you'll need to create a new file, or files, which contain each of the alter statements and these are the *real* migration steps. The schema.cql above is a reified example, but it's not the actual one used by customers. 

so consider adding a file like `timestamps.cql`

and in it something like
```
ALTER TYPE executions ADD created timestamp;
ALTER TYPE executions ADD updated timestamp;

ALTER TYPE history_node ADD created timestamp;

ALTER TYPE history_tree ADD created timestamp;

...
```